### ISSUE TITLE
Unit has to be specified when calling makeAspectCrop

### ISSUE THREAD

**Issue by Englund0110** [DominicTobias/react-image-crop#270]
Hey!

When calling makeAspectCrop(), i have to specify the unit to be either "px" or "%" in the crop object.

If i call makeAspectCrop() with this Crop object: 
```json
{
    aspect: 0.7776332899869961,
    height: 151,
    width: 116,
    x: 37,
    y: 88
}
```

without specifying the unit, it returns  
```json
{
    aspect: 0.7776332899869961,
    height: 12,
    width: 12,
    x: 37,
    y: 88,
    unit: "px"
}
```

If i specify the unit to be "px" like this
```json
{ 
    aspect: 0.7776332899869961,
    height: 151,
    width: 116,
    x: 37,
    y: 88,
    unit: "px" 
}
``` 
it returns 
```json
{
    aspect: 0.7776332899869961,
    height: 149.1705685618729,
    width: 115.99999999999999,
    x: 37,
    y: 88,
    unit: "px"
}
```
 which is correct. 

Shouldn't it default be "px"?

**Comment by DominicTobias** [comment]
Hi thanks you are right, will fix later

**Comment by Englund0110** [comment]
Sounds great!

**Comment by DominicTobias** [comment]
Should be fixed in 8.0.3, let me know if you have any probs

**Comment by Aanchal1103** [comment]
I think the `unit` returned should have been changed to `%` instead of the height width values.
It broke our site (and maybe others too) because the values being returned, changed.
